src/Honeycomb.OpenTelemetry/MeterProviderBuilderExtensions.cs

```diff
@@ -25,9 +27,17 @@ public static class MeterProviderBuilderExtensions
         /// </summary>
         public static MeterProviderBuilder AddHoneycomb(this MeterProviderBuilder builder, HoneycombOptions options)
         {
+            options.ApplyEnvironmentOptions(new EnvironmentOptions(Environment.GetEnvironmentVariables()));
+
+            if (options is null)
+            {
+                throw new ArgumentNullException(nameof(options), "No Honeycomb options have been set in appsettings.json, environment variables, or the command line.");

```

Comment:
We don't support command line args anymore 😄 
```suggestion
                throw new ArgumentNullException(nameof(options), "No Honeycomb options have been set in appsettings.json or environment variables.");
```